README.md

```diff
@@ -273,9 +278,21 @@ spec:
     image: your-customized-auto-instrumentation-image:python
 ```
 
-The Dockerfiles for auto-instrumentation can be found in [autoinstrumentation directory](./autoinstrumentation). 
+The Dockerfiles for auto-instrumentation can be found in [autoinstrumentation directory](./autoinstrumentation).
 Follow the instructions in the Dockerfiles on how to build a custom container image.
 
+#### Inject OpenTelemetry SDK environment variables only
+
+You can configure the OpenTelemetry SDK for applications which can't
+currently be autoinstrumented by using `inject-sdk` in place of (e.g.)
+`inject-python` or `inject-java`. This will inject environment variables
+like `OTEL_RESOURCE_ATTRIBUTES` but will not actually provide the SDK via an
```

Comment:
> via an init container

This is an implementation detail and it will not be true for e.g. Golang instrumentation. therefore it should not be mentioned here.